hack/release-notes.tmpl

```diff
@@ -1,3 +1,5 @@
+{{ .CurrentRevision}}
```

Comment:
This is redundant imo. It is already a part of the headline (the tag). But since this is only keeping the existing behavior, I don't mind keeping it and changing min a different PR. Not a big deal